data/processors/export.py

```diff
@@ -40,6 +40,18 @@ def unlocalise(value: str | list[Any] | dict[str, Any]) -> Any:
         return {k: unlocalise(v) for k, v in value.items()}
     raise ValueError(f"Unhandled type {type(value)}")
 
+def normalise_id(_id: str) -> str:
+    """removes leading zeros from all point-separated parts of input string"""
+    parts=_id.split(".")
+    for i in range(0, len(parts)):
+        parts[i]=parts[i] .lstrip('0')
+        if not parts[i]:
+            parts[i]="0"
+    result: str = "";
+    for part in parts:
+        result+=part;
+    return ".".join(parts)
+
```

Comment:
(not a nit by me, but rather how ruff wants this formatted)

```suggestion

def normalise_id(_id: str) -> str:
    """Remove leading zeros from all point-separated parts of input string"""
    parts = _id.split(".")
    for i in range(0, len(parts)):
        parts[i] = parts[i].lstrip("0")
        if not parts[i]:
            parts[i] = "0"
    result: str = ""
    for part in parts:
        result += part
    return ".".join(parts)

```